letshelp-letsencrypt/setup.py

```diff
@@ -0,0 +1,22 @@
+import sys
+
+from setuptools import setup
+from setuptools import find_packages
+
+
+install_requires = []
+if sys.version_info < (2, 7):
+    install_requires.append("mock<1.1.0")
+else:
+    install_requires.append("mock")
+
+setup(
+    name="letshelp-letsencrypt",
+    packages=find_packages(),
+    install_requires=install_requires,
+    entry_points={
+        'console_scripts': [
+            "letshelp-letsencrypt-apache = letshelp_letsencrypt.apache:main",
+        ],
+    },
+)
```

Comment:
Please add `include_package_data=True`. Otherwise we'll face similar issues #634 has fixed for apache and nginx plugins.